bangazonreports/views/users/favorited_sellers.py

```diff
@@ -0,0 +1,51 @@
+import sqlite3
+from django.shortcuts import render
+from bangazonapi.models import Product
+from bangazonreports.views import Connection
+
+
+def favorited_sellers(request):
+    if request.method == 'GET':
+        with sqlite3.connect(Connection.db_path) as conn:
+            conn.row_factory = sqlite3.Row
+            db_cursor = conn.cursor()
+            db_cursor.execute("""
+              SELECT
+	              cust.first_name || cust.last_name AS customer_name,
+                fav.first_name || fav.last_name AS favorite_name
```

Comment:
The `customer_name` and `favorite_name` are printing out in the html without a space between them

Reply:
Oops, yes I'll fix and merge